bulbs/special_coverage/management/commands/migrate_active_to_published.py

```diff
@@ -0,0 +1,10 @@
+from django.core.management.base import BaseCommand
+
+from ...utils import publish_active_special_coverages
+
+
+class Command(BaseCommand):
+    help = "Migrates Special Coverages to published"
+
+    def handle(self, *args, **kwargs):
+        publish_active_special_coverages()
```

Comment:
any reason this is in a separate module? I can't see this being used outside of the 1-time migration command